urbanopt/geojson-modelica-translator
# refactor: add template source to generated files

~~⚠️ WIP ⚠️ 
The target merge branch should be resolved first before we merge this. This is b/c we have to diff a model updated by @helarga, which this PR would make messier.~~

#### Any background context you want to provide?
Currently, the GMT doesn't provide any information about where generated modelica code comes from. This can make debugging/iterating on model changes very difficult when looking at only the generated codde.

#### What does this PR accomplish?
Adds comments to the generated DistrictEnergySystem about what model the section is for, and what template file was used to generate it.

#### How should this be manually tested?
Run one of the tests, verify that the District/DistrictEnergySystem.mo has these comments

#### What are the relevant tickets?
#### Screenshots (if appropriate)
<img width="616" alt="Screen Shot 2021-01-22 at 1 28 39 PM" src="https://user-images.githubusercontent.com/18518728/105541910-bf1b3780-5cb5-11eb-9f4b-1747cca86d95.png">
<img width="903" alt="Screen Shot 2021-01-22 at 1 29 26 PM" src="https://user-images.githubusercontent.com/18518728/105541983-da864280-5cb5-11eb-9e49-812112025094.png">
